packages/odata-common/test/common-entity.ts

```diff
@@ -0,0 +1,147 @@
+/* eslint-disable max-classes-per-file */
```

Comment:
Question: Is this file is auto generated?
Or will it be?

Reply:
no, the idea was to implement a entity based on the base which is not a v2 or v4 entity to test the content which is shared between oData versions.